lib/datadog/kit/events.rb

```diff
@@ -7,41 +7,62 @@ module Datadog
   module Kit
     # Tracking events
     module Events
-      # Attach login event information to the trace
+      APPSEC_LOGIN_SUCCESS_EVENT = 'users.login.success'
+      APPSEC_LOGIN_FAILURE_EVENT = 'users.login.failure'
+
+      # Attach login success event information to the trace
+      #
+      # This method is experimental and may change in the future.
```

Comment:
I think this approach of having something and declaring it experimental is fine BUT should we be more specific on what users can expect?

E.g. does this mean that an API added on 1.9 can be removed on 1.10? Or would we just deprecate and no-op it?

Reply:
It won't be removed but the signature may change, and maybe the method names or namespace, but hopefully not, I'm trying to nail that down before merging.

Reply:
I think any of the ones you mentioned are fine, as long as we document that (if I'm understanding what you're saying) it can break even between dd-trace-rb minor versions -- just so customers that pick it up are aware of this possibility.

Reply:
Hmmm, maybe we could add a YARD doc annotation like we did for `@public_api`? `@experimental`?

Alternatively, if the user wouldn't call this method directly, it may be worth privatizing until it's stable.

Reply:
Like `Identity::set_user`, this is a sugary public API to be used by customers to help them annotate traces with a higher level of abstraction. It can't be made private.

Regarding `@experimental`, I would love that, although it sounds a bit scary: since we are going to release that as a public beta feature I would love to have a `@beta` annotation as well.

To me the difference would be:

- experimental: "we're attempting something that you can try if you find it useful, but we're exploring something, so all bets are off and it might change in any shape or form, or may very well vanish in the future"
- beta: "this is something that's not quite stable yet, it will probably not change much but it still may, and will most likely _not_ vanish from under your feet"